main.py

```diff
@@ -218,9 +218,9 @@ def ensure_driver_image(root_path, branch_name, driver_name):
     os.makedirs(neo4jArtifactsPath)
 
     neo4jServers = [
-        { "version": "3.5", "edition": "enterprise", "cluster": False, "suite": "3.5", "scheme": "bolt" },
         { "version": "4.0", "edition": "community", "cluster": False, "suite": "4.0", "scheme": "neo4j" },
         { "version": "4.1", "edition": "enterprise", "cluster": False, "suite": "4.1", "scheme": "neo4j" },
+        { "version": "4.1", "edition": "enterprise", "cluster": False, "suite": "4.3", "scheme": "neo4j" }
```

Comment:
We want this soon but for now it will just take extra time and resources from the build environment. We should add  this when there are docker images available for 4.3 (not  necessariliy via dockerhub)